css/style.css

```diff
@@ -1015,9 +1000,182 @@ Desktop Site
 ===========================
 */
 
-@media (max-width: 768px) {
+@media screen and (max-width: 375px) {
+  body {
+    overflow-x: hidden;
+  }
+
   .desktop-nav {
-    display: none;
+    position: absolute;
+    display: flex;
+    right: 0;
+    height: 660.7px;
+    top: 0;
+    width: 100%;
+    background: #e5e5e5;
+    flex-direction: column;
+    align-items: center;
+    z-index: 2;
+    transform: translateX(100%);
+    transition: transform 0.5s ease-in;
+  }
+
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![pppppppp2](https://user-images.githubusercontent.com/79058364/162807217-42958bb4-febc-4035-a494-3ed9fa9cff9e.png) | ![pppppppppppp1](https://user-images.githubusercontent.com/79058364/162807295-c4661eda-4197-46ab-ab3a-7efad86e2e31.png)


- [OPTIONAL ] What if you use `position: fix` instead of `absolute` ? That will help preventing your mobile navigation menu to scroll, and don't forget to give the the mobile navigation menu the height of 100% of the page  as on the figma design 


- [OPTIONAL ] It would be better for the background not to be scrollable when the mobile nav menu is open . you can use the CSS property `overflow: hidden;` to prevent that.

- [ OPTIONAL] I think  it will be better as well to close the mobile navigation menu page once you click  on of the mobile menu option.



- [ ]  Adding these features will make your design look good and look exactly like Figma design, I highly recommend you do this in the meantime, I'll approve this project since the previous reviewer is skipping this

Reply:
Thanks for the review. I'll get on the recommendations you've made.